plugins/kotlin/j2k/k1.new/src/org/jetbrains/kotlin/nj2k/printing/JKCodeBuilder.kt

```diff
@@ -65,7 +66,8 @@ internal class JKCodeBuilder(context: NewJ2kConverterContext) {
         }
 
         private fun renderModifiersList(modifiersListOwner: JKModifiersListOwner) {
-            val isInterface = modifiersListOwner is JKClass && modifiersListOwner.classKind == INTERFACE
+            val isInterface = (modifiersListOwner is JKClass && modifiersListOwner.classKind == INTERFACE) ||
```

Comment:
I would rename it to `isOpenByDefault` or smth similar because now it's `true` not only for interfaces.